lib/index.js

```diff
@@ -2,6 +2,11 @@
 
 const getSpecificationObject = require('./helpers/getSpecificationObject');
 
+const createSpecification = require('./helpers/createSpecification');
+const parseApiFileContent = require('./helpers/parseApiFileContent');
+const updateSpecificationObject = require('./helpers/updateSpecificationObject');
+const finalizeSpecificationObject = require('./helpers/finalizeSpecificationObject');
```

Comment:
No need to import these here and export them later when they are not used in this file. `getSpecificationObject()` is the only one actually used here, the rest are to be required and used whenever and wherever they are actually needed, i.e. https://github.com/Surnet/swagger-jsdoc/pull/170/files#diff-63a921b666f863ffc9f3199ae07ca772

Reply:
It's actually so users are able to import them like this

```js
const {
  createSpecification,
  updateSpecificationObject,
} = require('swagger-jsdoc');
```

Otherwise you would need to do this
```js
const createSpecification = require('swagger-jsdoc/lib/createSpecification');
```

And in this case you make your file tree a public API, meaning that moving/renaming a file would be a breaking change. Import/export in the main module file is actually done by a lot of packages (check out this one by [redux-form](https://github.com/erikras/redux-form/blob/master/src/index.js)), and since they are already dependencies of `getSpecificationObject` anyway, it does not make the require tree bigger.

I can change for sure, just let me know 😃.

Reply:
Cool, I didn't consider this, please leave it like this, of course! :)
It's good then, if you like to add some notes and a link to your plugin, please do, if now I can merge and publish :)
Thank you!

Reply:
I added links for my plugin and another one that rebuilds with webpack but using a list of predefined paths (like this package). Thanks !